lib/crawler/keyword/keywords.ex

```diff
@@ -10,7 +10,14 @@ defmodule Crawler.Keyword.Keywords do
   alias Crawler.Repo
   alias CrawlerWorker.Keyword.CrawlerWorker
 
-  def list_keywords(user_id), do: Repo.all(KeywordQuery.user_keywords_query(user_id))
+  def list_keywords_by_filter_params(user_id, filter_params \\ %{}) do
```

Comment:
How about

```suggestion
  def list_user_keywords_by_filter_params(user_id, filter_params \\ %{}) do
```

Reply:
Fixed in [6bd2542](https://github.com/vnntsu/elixir-ic/pull/44/commits/6bd254207c503d8f6476d6ca4c09ee1f952d6dbc)